react-components/src/components/Checkout/editableForm.js

```diff
@@ -64,7 +68,15 @@ const EditableForm = props => {
 
     const handleSubmitAddressForm = useCallback(
         formValues => {
-            setShippingAddressesOnCart({ variables: { cartId: cartId, countryCode: 'US', ...formValues } });
+            setShippingAddressesOnCart({ variables: { cartId: cartId, countryCode: 'US', ...formValues } })
+                .catch(error => {
+                    dispatch({ type: 'error', error: error.toString() });
+                })
+                .finally(() => {
+                    dispatch({ type: 'endLoading' });
```

Comment:
The `beginLoading` and `endLoading` events are only available via the `CartContext`, so you need to use `cartDispatch` here.

Reply:
Thanks, I updated the events to use cartDispatch here.

Reply:
@dabe5651 please make sure you run tests locally first and they pass. CircleCI reports test failures.